practice/case.py

```diff
@@ -0,0 +1,138 @@
+""" Takes input in CamelCase, CONSTANT_CASE, kebab-case, or snake_case and converts to any of the others."""
```

Comment:
Look at the example docstrings in [PEP-257](https://www.python.org/dev/peps/pep-0257/). They have no spaces before the start of the sentence, the summary sentence ends with a period, they have closing triple-double quotes only on the next line if the docstring is multiple lines. Ensure that your docstrings look like that.